container/container.go

```diff
@@ -230,3 +232,65 @@ func deleteEmpty(s []string) []string {
 	}
 	return r
 }
+
+func SeccompEnforcingMode() string {
+
+	// Try to read from /proc/self/status for kernels > 3.8
+	s, err := parseStatusFile()
+	if err != nil {
+		// Check if Seccomp is supported, via CONFIG_SECCOMP.
+		if err := unix.Prctl(unix.PR_GET_SECCOMP, 0, 0, 0, 0); err != unix.EINVAL {
+			// Make sure the kernel has CONFIG_SECCOMP_FILTER.
+			if err := unix.Prctl(unix.PR_SET_SECCOMP, unix.SECCOMP_MODE_FILTER, 0, 0, 0); err != unix.EINVAL {
+				return "strict"
+			}
+		}
+		return "disabled"
+	}
+
+	seccomp_modes := map[string]string{
+		"0": "disabled",
+		"1": "strict",
+		"2": "filtering",
+	}
+
+	seccompMode, ok := s["Seccomp"]
+	if !ok {
+		return "undefined"
+	}
+
+	t, ok := seccomp_modes[strings.TrimSpace(seccompMode)]
+	if !ok {
+		return "undefined"
+	}
+
+	return t
+}
+
+func parseStatusFile() (map[string]string, error) {
```

Comment:
So I have a function already that reads a file can probably just reuse that and then do a strings.Contains instead so as to avoid another utility function :)